README-cn.md

```diff
@@ -0,0 +1,215 @@
+# Nano ID
+
+<img src="https://ai.github.io/nanoid/logo.svg" align="right"
+     alt="Nano ID logo by Anton Lovchikov" width="180" height="94">
+
+一个用于 JavaScript 的唯一字符串 ID 生成器，它迷你、安全，并且对 URL 友好。
+
+**安全：** It uses cryptographically strong random APIs
```

Comment:
Should we translate it too?

Reply:
Oh yes, actually I've translated that, but forgot to delete the English version of it.